cflib/drivers/cfusb.py

```diff
@@ -29,6 +29,8 @@
 import os
```

Comment:
`os` is not used anymore?

Reply:
Is is now re-used thanks to handling special case for the special Windows ...